docs/guides/background-fetching-indicators.md

```diff
@@ -2,21 +2,17 @@ A query's `status === 'loading'` state is sufficient enough to show the initial
 
 To do this, queries also supply you with an `isFetching` boolean that you can use to show that it's in a fetching state, regardless of the state of the `status` variable:
 
+?> The example below uses `<script setup>` syntax.
```

Comment:
Do you think that it would be better to add a section to the `installation` page explaining how to setup `compositionAPI` and `script setup` for Vue2, and maybe link this warning to there, or remove it altogether?

Reply:
Yep, will write a section in the `installation` page and remove the warning

Reply:
Done